chainnotif/neutrinonotify/driver.go

```diff
@@ -25,8 +25,8 @@ func createNewNotifier(args ...interface{}) (chainnotif.ChainNotifier, error) {
 
 	spendHintCache, ok := args[1].(chainnotif.SpendHintCache)
 	if !ok {
-		return nil, errors.New("second argument to neutrinonotify.New " +
-			"is  incorrect, expected a chainntfs.SpendHintCache")
+		return nil, errors.New("second argument to neutrinonotify.New" +
```

Comment:
We should always have the space within wrapped error or log strings on the previous line, makes review easier. So if the space makes things too long, I'd just break a word earlier... Same below.